packages/manager/modules/pci/src/projects/project/ai-dashboard/ai-dashboard.service.js

```diff
@@ -153,4 +153,34 @@ export default class AiDashboardService {
       )
       .then(({ data }) => data);
   }
+
+  getRegistry(projectId, registryId) {
+    return this.$http
+      .get(
+        `/cloud/project/${projectId}/ai/registry/${registryId}`,
+        AiDashboardService.getIcebergHeaders(),
```

Comment:
**question:** Is it necessary to make this an `iceberg` call?

Reply:
Hi Anoop, seems to be ok in local, so I pushed it but I will test in staging to make sure.

Reply:
Seems to work without Iceberg call in staging! Thx :)